README.md

```diff
@@ -6,11 +6,30 @@ to allow codebases already using AWS credentials to use IAM roles with minimal t
 refactoring. In the same vein, the Role Injector can also be used to help users using the
 command line tools to assume a role.
 
+## Assumptions
+
+ - At least two AWS Accounts:
+   - AWS Account 1 must have a policy that includes sts:AssumeRole to AWS Account 2
+   - AWS Account 2 must have a Trust Relationship on a role that references AWS Account 1
```

Comment:
This is not strictly necessary. Even if you have one AWS account, you could use this to maintain very restricted IAM permissions for non-MFA authenticated requests, and then assume a role in that same account to do more sensitive work.